content/articles/activity-tracking.markdown

```diff
@@ -0,0 +1,49 @@
+---
+title: Activity Tracking
+excerpt: How to manage the users on a DNSimple account.
+categories:
+- Account
+---
+
+# Managing account users
+
+### Table of Contents {#toc}
+
+* TOC
+{:toc}
+
+---
+
+<note>
+Personal plan subscribers have access to all events that occurred during the last 30 days. A [Professional and Business subscription](https://dnsimple.com/pricing) is required to view the full history.
+</note>
+
+Activity tracking provides an overview of all past changes for a specific domain or for an account as a whole.
+
+## Activity tracking at the domain level
+
+Domain level activity tracking can be found under the domain page.
```

Comment:
In screenshot 1 I'd show a larger section, as you did for the account, to help understand where it is located. The "domain page" is vague, we should indicate in which tab and where precisely.

Having a screenshot that shows its position would help as well.